src/Controls/src/Core/Compatibility/Handlers/Shell/iOS/SearchHandlerAppearanceTracker.cs

```diff
@@ -165,6 +165,7 @@ void UpdateSearchBarBackgroundColor(UITextField textField)
 			if (_defaultBackgroundColor == null)
 				_defaultBackgroundColor = backgroundView.BackgroundColor;
 			backgroundView.BackgroundColor = backGroundColor.ToPlatform();
+			textField.BackgroundColor = backGroundColor.ToPlatform();
```

Comment:
can't this `backGroundColor` be cached at one filed and reused? Something like

```suggestion
			var backgroundColor = backGroundColor.ToPlatform();
			backgroundView.BackgroundColor = backgroundColor;
			textField.BackgroundColor = backgroundColor;
```